### ISSUE TITLE
Bus Error: 10 - Onset and frames

### ISSUE THREAD

**Issue by fenrirx22** [tensorflow/magenta#1346]
Hello, I've installed the magenta and created a note sequence and played it from python code, just to test things out. After that I've tried to transcribe a small wav file (around 1mb ) with onset and frames model and got the "Bus error: 10" in the console :

```
2018-11-14 16:39:42.804463: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
INFO:tensorflow:Restoring parameters from ../resources/train/model.ckpt-100154
INFO:tensorflow:Starting transcription for ../resources/sample.wav...
INFO:tensorflow:Processing file...
INFO:tensorflow:Running inference...
Bus error: 10
```

this is the result from the mac user console -> user reports -> https://pastebin.com/eUpP59qt

Running on  MacOS high sierra 10.13.3
cpu: i5 2,3gHz
ram: 16 GB 2133 MHz LPDDR3

**Comment by cghawthorne** [comment]
@fenrirx22 I'm not sure what would cause that error, but I suspect it's something with your Python or Tensorflow setup, since all our code is in Python and shouldn't be capable of causing bus errors. Can you try reinstalling Tensorflow, perhaps without GPU support and see if that helps? If not, this might be a bug report for the Tensorflow issue tracker.

**Comment by fenrirx22** [comment]
Ok,  After reinstalling almost everything ( including python 2.7.15 and 3.6.5 ) I've found the solution! I've checked all the versions of tensorflow/magenta/python used in the docker image. It seems like the newest tensorflow ver 1.12.0 which is installed automatically (via pip install tensorflow) produces this problem. Docker image uses ver 1.10.0 (pip install tensorflow==1.10.0) so I've installed the previous version and it started working :)
